File: Backend/src/main/java/com/feedle/feedleapi/Networking/DeleteFriendRequest.java
package com.feedle.feedleapi.Networking;

public class DeleteFriendRequest extends Request {
    private int userFriendId;
    public DeleteFriendRequest(int userFriendId)
    {
        super(RequestType.DeleteFriendRequest);
        this.userFriendId = userFriendId;
    }

    public int getUserFriendId() {
        return userFriendId;
    }
}

File: FeedleDataTier/FeedleDataTier/Network/DeleteFriendRequest.cs
using System.Text.Json.Serialization;

namespace FeedleDataTier.Network
{
    public class DeleteFriendRequest : Request
    {
        [JsonPropertyName("userFriendId")]
        public int UserFriendId { get; set; }

        public DeleteFriendRequest(int userFriendId) : base(RequestType.DeleteFriendRequest)
        {
            this.UserFriendId = userFriendId;
        }
    }
}